scripts/prepare_rirs.py

```diff
@@ -274,33 +432,106 @@ def prepare_arni(path_raw, path_sofa, formats=["mic", "foa"]):
         print(f"Finished .sofa creation for {aud_fmt} format.")
 
 
+def prepare_daga(source_path, sofa_path, audio_fmts=["mic"]):
+
+    # microphone assumed to be in the center
+    mic_position = np.array([0, 0, 0])
+
+    # the source positions are exactly in front of the microphone
+    source_positions = {"0": np.array([2.5, 0, 0]), "180": np.array([2.8, 0, 0])}
+
+    sofa_folder = Path(source_path)
+    dest_path_sofa = Path(sofa_path)
+    aggregated_irs = {fmt: [] for fmt in audio_fmts}
+    aggregated_source_positions = {fmt: [] for fmt in audio_fmts}
+
+    for sofa_file in sofa_folder.glob("*.sofa"):
+
+        source_angle = "180" if "180" in sofa_file.name else "0"
+        source_pos = source_positions[source_angle] + random.uniform(-0.001, 0.001)
+
+        sofa = netCDF4.Dataset(sofa_file, "r")
+        irs = sofa.variables["Data.IR"][:]
+        orig_sr = sofa.variables["Data.SamplingRate"][:][0]
+        sofa.close()
+
+        for fmt in audio_fmts:
+            if fmt == "mic":
+                processed_irs = irs[:, __TETRA_CHANS_IN_EM32__, 0].T
+            else:
+                # place array2sh conversion here
```

Comment:
is this `array2sh` done? just making sure that you don't forget to add it here in case it is. hehe

Reply:
thank you! we are working on it, but it won't be part of this PR yet.